cmake/AlembicPyIlmBase.cmake

```diff
@@ -33,16 +33,26 @@
 ##
 ##-*****************************************************************************
 
-FIND_PACKAGE(Imath)
-
 SET(PYTHON_VERSION_MAJOR ${PYALEMBIC_PYTHON_MAJOR})
 SET(PYTHON_VERSION_MINOR ${Python${PYALEMBIC_PYTHON_MAJOR}_VERSION_MINOR})
 SET(PYTHON_INCLUDE_DIR ${Python${PYTHON_VERSION_MAJOR}_INCLUDE_DIRS})
 
+
+FIND_PACKAGE(Imath COMPONENTS PyImath)
 IF (Imath_FOUND)
-    SET(ALEMBIC_PYILMBASE_PYIMATH_LIB Imath::PyImath_Python${PYTHON_VERSION_MAJOR}_${PYTHON_VERSION_MINOR})
-    MESSAGE(STATUS "Found package Imath using: ${ALEMBIC_PYILMBASE_PYIMATH_LIB}")
+    SET(ALEMBIC_PYILMBASE_PYIMATH_LIB Imath::PyImath)
+    MESSAGE(STATUS "Found package PyImath using: ${ALEMBIC_PYILMBASE_PYIMATH_LIB}")
 ELSE()
+    FIND_PACKAGE(Imath)
+
+    IF (Imath_FOUND)
```

Comment:
It's possible here that `find_package(Imath)` succeeded but the libPyImath library wasn't a part of it. Should ypu validate that and possibly report an informative error? As is, it will just continue on with a reference to a library that doesn't exist and error out. Which is maybe OK?

Glad this worked, BTW!

Reply:
This is was the original behavior, left in for now in case people need to try older versions of 3.1 or 3.0 before PyImath was exposed as a component.